graypy/handler.py

```diff
@@ -379,36 +372,36 @@ def send(self, s):
 
 
 class GELFTCPHandler(BaseGELFHandler, SocketHandler):
-    """Graylog Extended Log Format TCP handler"""
+    """GELF TCP handler"""
 
     def __init__(self, host, port=12201, **kwargs):
         """Initialize the GELFTCPHandler
 
         :param host: The host of the Graylog server.
```

Comment:
Maybe document as  "GELF TCP input host"? Using a similar format as to what is documented by the `GELFHTTPHandler`